src/components/layout/style.module.scss

```
@@ -0,0 +1,10 @@
+@import '../../styles/vars';
```

Why don't you try to use this?
```suggestion
@import 'styles/vars';
```

As you are using NODE_SASS .env variable I think is possible to change it everywhere